tests/std/tests/P1004R2_constexpr_vector_bool/test.cpp

```diff
@@ -620,6 +620,52 @@ constexpr bool test_iterators() {
     return true;
 }
 
+#ifndef __EDG__ // TRANSITION, VSO_1274387
+constexpr bool test_gh2440() { // COMPILE-ONLY

```

Comment:
Could we move that to the Tests around line 400 instead?

This is basic functionality

Reply:
IIUC, you mean to insert (pun intended) this as a normal "insert a non-forward range" test case - which we are missing - among the rest of the test coverage. Yes, this makes sense to me. I'll move it and rephrase the comment into an "this also guards against regression of GH-2440" comment.

Reply:
Thanks, and appologies for not covering input ranges.

You know we could also construct from an input range 😇